guide/popular-topics/permissions.md

```diff
@@ -324,6 +324,34 @@ If you want to know how to work with the returned Permissions objects keep readi
 The <branch version="11.x" inline>[Permissions](https://discord.js.org/#/docs/main/v11/class/Permissions)</branch><branch version="12.x" inline>[Permissions](https://discord.js.org/#/docs/main/stable/class/Permissions)</branch> object is a discord.js class containing a permissions bit field and a bunch of utility methods to manipulate it easily.
 Remember that using these methods will not manipulate permissions, but create a new instance representing the changed bit field.
 
+### Displaying permissions as an array
+
+Discord.js provides a `toArray()` method, which can be used to convert a `Permissions` object into an array containing permission flags. This is useful if you want to display/list them and it enables you to use other array methods. For example:
```

Comment:
- Seeing that #toArray is mostly used for debugging and pretty-printing permissions it should not be at the end of this section, rather than the beginning before even explaining the constructor
- can explain both `.toArray()` as well as `.serialize()` under the heading "Displaying permission flags" or similar

Reply:
yup, added that